install/helm/agones/templates/crds/_gameserverspecvalidation.yaml

```diff
@@ -148,4 +148,30 @@ properties:
             type: integer
             minimum: 1
             maximum: 2147483648
+      alpha:
+        type: object
+        title: Alpha properties for the GameServer
+        properties:
+          players:
+            type: object
+            title: Configuration of player capacity
+            properties:
+              initialCapacity:
+                type: integer
+                title: The intial player capacity that this Game Server has
+                minimum: 0
+              webhook:
```

Comment:
Do we need to add a CABundle as we have for FAS Webhook Policy?
https://agones.dev/site/docs/reference/agones_crd_api_reference/#WebhookPolicy

Reply:
That's an excellent question. I copied what we have in https://github.com/googleforgames/agones/blob/master/install/helm/agones/templates/crds/fleetautoscaler.yaml - and there is no reference to a caBundle. I assume there should be?

Reply:
Thanks for noticing this, definitely it should be.

Reply:
I've implemented it here (awaiting to push up), I think caBundle should be added to fleetautoscaler validation in a separate PR, just to keep things separate. Sound good?

Reply:
Thanks, now I see it.